config.yml

```
@@ -1,4 +1,5 @@
 jtype: GlbCrafter
 metas:
   py_modules:
+    - utils.py
```

This structure is prefered,
```
.
├── config.yml
└── executor
    ├── util.py
    ├── __init__.py
    └── glb_crafter.py
```

```suggestion
   - executor/__init__.py
```